### PR TITLE
Feature/40324

### PR DESCRIPTION
**Pull request by suraj-aggarwal**
[issue no. 40324] (https://redmine.successive.tech/issues/40324)
 Day 2 - HTML Input and Images

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TextFieldDemo from './pages/index'; 
+function App() {
+  return (
+    <>
```
Comment (madhavBansal-successiveTech) [inline src/App.jsx:5]: Remove unwanted fragments

File: src/components/TextField/TextField.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input } from './style';
+
+function TextField(props) {
+  console.log('--------Inside TextField Constructor---------', props);
```
Comment (madhavBansal-successiveTech) [inline src/components/TextField/TextField.jsx:6]: remove console.log

File: src/components/TextField/TextField.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+TextField.propTypes = {
+  value: PropTypes.string,
+  disabled: PropTypes.bool,
+  error: PropTypes.string,
+};
+
+TextField.defaultProps = {
+  value: '',
+  disabled: false,
```
Comment (madhavBansal-successiveTech) [inline src/components/TextField/TextField.jsx:24]: use default value true for disable

File: src/components/TextField/TextField.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import PropTypes from 'prop-types';
+import { Input } from './style';
+
+function TextField(props) {
+  console.log('--------Inside TextField Constructor---------', props);
+  const { value, disabled, error } = props;
+  return (
+    <div>
+      <Input type="text" value={value} disabled={disabled} />
+      <p style={{ color: 'red' }}>{error}</p>
```
Comment (madhavBansal-successiveTech) [inline src/components/TextField/TextField.jsx:11]: use color value from style.js and only shows error tag if error is occured

File: src/components/TextField/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export { TextField as default } from './TextField';
```
Comment (madhavBansal-successiveTech) [inline src/components/TextField/index.js:1]: use -  export { default as TextField } from './TextField';

File: src/components/TextField/TextField.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  error: PropTypes.string,
+};
+
+TextField.defaultProps = {
+  value: '',
+  disabled: false,
+  error: '',
+};
+
+export { TextField };
```
Comment (madhavBansal-successiveTech) [inline src/components/TextField/TextField.jsx:28]: export this as default - 
export default TextField

File: src/pages/TextFieldDemo/TextFieldDemo.jsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { TextField } from '../../components/index';
+
+function TextFieldDemo() {
+  return (
+    <>
+      <p> The disabled input</p>
+      <br />
```
Comment (madhavBansal-successiveTech) [inline src/pages/TextFieldDemo/TextFieldDemo.jsx:8]: try to avoid "br"

File: src/pages/TextFieldDemo/index.js [hunk 8]
```diff
@@ -0,0 +1 @@
+export {default as TextFieldDemo} from './TextFieldDemo';
```
Comment (madhavBansal-successiveTech) [inline src/pages/TextFieldDemo/index.js:1]: fix linting error for must a new line at end of file

**Review by madhavBansal-successiveTech — COMMENTED** [review]
Remove all unwanted file like serviceWorker.js , App.test.jsx , theme.js etc

**Comment by suraj-aggarwal** [comment]
pr closed changing pr.

File: src/components/TextField/TextField.jsx (hunk 2 above)
Comment (suraj-aggarwal) [reply src/components/TextField/TextField.jsx:6]: Done

File: src/components/TextField/TextField.jsx (hunk 3 above)
Comment (suraj-aggarwal) [reply src/components/TextField/TextField.jsx:24]: Done

File: src/App.jsx (hunk 1 above)
Comment (suraj-aggarwal) [reply src/App.jsx:5]: Done

File: src/components/TextField/TextField.jsx (hunk 4 above)
Comment (suraj-aggarwal) [reply src/components/TextField/TextField.jsx:11]: Done

File: src/components/TextField/index.js (hunk 5 above)
Comment (suraj-aggarwal) [reply src/components/TextField/index.js:1]: Done

File: src/pages/TextFieldDemo/index.js (hunk 8 above)
Comment (suraj-aggarwal) [reply src/pages/TextFieldDemo/index.js:1]: Done

File: src/pages/TextFieldDemo/TextFieldDemo.jsx (hunk 7 above)
Comment (suraj-aggarwal) [reply src/pages/TextFieldDemo/TextFieldDemo.jsx:8]: Done

**Pull request closed without merging**